.travis.yml

```diff
@@ -23,17 +22,36 @@ cache:
   cargo: true
 before_script:
   - (which cargo-install-update && cargo install-update cargo-update) || cargo install cargo-update
-  - (which rustfmt && cargo install-update rustfmt) || cargo install rustfmt
   - (which cargo-prune && cargo install-update cargo-prune) || cargo install cargo-prune
+  - if [ "${TRAVIS_RUST_VERSION}" = stable ]; then
+      (which rustfmt && cargo install-update rustfmt) || cargo install rustfmt;
+    elif [ "${TRAVIS_OS_NAME}" = linux ]; then
+      clippy_vers=0.0.99;
```

Comment:
We must update this to `0.0.120` and nightly to `2017-03-16`, no ? If we do `cargo clippy --profile=test` (and not `cargo clippy`) we should be fine - that's the workaround for the moment for `safe_authenticator/safe_app` crates too in the `dev` branch of safe_client_libs if you want to refer.